generators/app/templates/actions.js

```diff
@@ -0,0 +1,3 @@
+export default function <%= name.replace('Actions', '') %> () {
+
+};
```

Comment:
What if we just returned an unnamed arrow function here? Either way, I think the note above will determine which route to go on this one too.